### DISCUSSION TITLE
node-chatgpt-api for Windows

### DISCUSSION THREAD

**Discussion by araea** [viopsa233/AI-testers-Forum#10] [技术交流]
<html>
<body>
<div class="lake-content" typography="classic"><details class="lake-collapse"><summary id="u5067fcd2"><strong><span class="ne-text">参考链接：</span></strong></summary><ol class="ne-ol" style="margin: 0; padding-left: 23px"><li id="u88cff022" data-lake-index-type="0"><a href="https://github.com/iouzzrj/node-chatgpt-apiTutorial/blob/master/bingTutorial.md" data-href="https://github.com/iouzzrj/node-chatgpt-apiTutorial/blob/master/bingTutorial.md" target="_blank" class="ne-link"><span class="ne-text">https://github.com/iouzzrj/node-chatgpt-apiTutorial/blob/master/bingTutorial.md</span></a></li><li id="u57ef1760" data-lake-index-type="0"><a href="https://github.com/waylaidwanderer/node-chatgpt-api" data-href="https://github.com/waylaidwanderer/node-chatgpt-api" target="_blank" class="ne-link"><span class="ne-text">https://github.com/waylaidwanderer/node-chatgpt-api</span></a></li></ol></details><details class="lake-collapse"><summary id="u351ad6e1"><strong><span class="ne-text">原Ai Testers中的大佬307的教程：</span></strong></summary><ol class="ne-ol" style="margin: 0; padding-left: 23px"><li id="u2624c5c5" data-lake-index-type="0"><span class="ne-text">首先需要你需要下载下来node-chatgpt-api</span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="u3720d411" data-lake-index-type="0"><a href="https://github.com/waylaidwanderer/node-chatgpt-api" data-href="https://github.com/waylaidwanderer/node-chatgpt-api" target="_blank" class="ne-link"><span class="ne-text">https://github.com/waylaidwanderer/node-chatgpt-api</span></a></li></ul><ol start="2" class="ne-ol" style="margin: 0; padding-left: 23px"><li id="uca9fd87a" data-lake-index-type="0"><span class="ne-text">然后你还需要下载一个node.js:  </span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="u13d66afe" data-lake-index-type="0"><a href="https://nodejs.org/en/download" data-href="https://nodejs.org/en/download" target="_blank" class="ne-link"><span class="ne-text">https://nodejs.org/en/download</span></a></li></ul><ol start="3" class="ne-ol" style="margin: 0; padding-left: 23px"><li id="uc50a4975" data-lake-index-type="0"><span class="ne-text">装好Node.js后 在“node-chatgpt-api”文件夹内打开终端 输入 </span><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">npm install</span></code></li><li id="u58b28be3" data-lake-index-type="0"><span class="ne-text">复制一份“setting.example.js”将其重命名为“settings.js”</span></li><li id="u3a2e4ac8" data-lake-index-type="0"><span class="ne-text">用记事本打开“settings.js”</span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="u81b7321d" data-lake-index-type="0"><span class="ne-text">Host不用填 userToken不用填</span></li><li id="u558d4789" data-lake-index-type="0"><span class="ne-text">cookies填入自己已经申请好了的newbing的账号cookies</span></li><li id="u54869c45" data-lake-index-type="0"><span class="ne-text">不知道获取cookies方法看补充  </span></li><li id="u5da58bb3" data-lake-index-type="0"><span class="ne-text">Proxy填入 &lt;http://127.0.0.1:xxxx&gt;（xxxx是梯子的端口号 记得本地也要打开vpn）</span></li></ul><ol start="6" class="ne-ol" style="margin: 0; padding-left: 23px"><li id="u6b5e17c5" data-lake-index-type="0"><span class="ne-text">把底下的port的地方改成3006</span></li><li id="u3bbd0c42" data-lake-index-type="0"><span class="ne-text">再把最后的那个//注释符删了</span></li><li id="u9d6bf607" data-lake-index-type="0"><span class="ne-text">保存退出  </span></li><li id="u7cb624a9" data-lake-index-type="0"><span class="ne-text">在当前目录下启动终端（cmd或者terminal） 输入 </span><code class="ne-code" style="font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, Courier, monospace; background-color: rgba(0, 0, 0, 0.06); border: 1px solid rgba(0, 0, 0, 0.08); border-radius: 2px; padding: 0px 2px"><span class="ne-text">npm run cli</span></code></li><li id="uf4f18938" data-lake-index-type="0"><span class="ne-text"> 一般就可以在命令行窗口和Sydney对话了</span></li></ol></details><details class="lake-collapse"><summary id="uf2cc512a"><strong><span class="ne-text">补充：</span></strong></summary><ol class="ne-ol" style="margin: 0; padding-left: 23px"><li id="u258c63a9" data-lake-index-type="0"><span class="ne-text">在node目录src目录下的BingAlClient.js那个文件里 可以更改“systemMessage”（设定）和“tuneStyle”（模式）</span></li><li id="u9e6b6f07" data-lake-index-type="0"><span class="ne-text">cookies获取方法（一个可能的方法）</span></li></ol><ul class="ne-ul" style="margin: 0; padding-left: 23px"><li id="uab8bc090" data-lake-index-type="0"><span class="ne-text">在电脑打开Edge 打开Microsoft Edge加载项 搜索并获取Tampermonkey </span></li><li id="u119df939" data-lake-index-type="0"><span class="ne-text"></span><a href="https://microsoftedge.microsoft.com/addons/detail/tampermonkey/iikmkjmpaadaobahmlepeloendndfphd" data-href="https://microsoftedge.microsoft.com/addons/detail/tampermonkey/iikmkjmpaadaobahmlepeloendndfphd" target="_blank" class="ne-link"><span class="ne-text">https://microsoftedge.microsoft.com/addons/detail/tampermonkey/iikmkjmpaadaobahmlepeloendndfphd</span></a></li><li id="u5f5a9a53" data-lake-index-type="0"><span class="ne-text">在Greasy Fork网站上搜索并安装“一键获取网站cookie” </span></li><li id="u53d96123" data-lake-index-type="0"><span class="ne-text"></span><a href="https://greasyfork.org/zh-CN/scripts/455411-getthecookie" data-href="https://greasyfork.org/zh-CN/scripts/455411-getthecookie" target="_blank" class="ne-link"><span class="ne-text">https://greasyfork.org/zh-CN/scripts/455411-getthecookie</span></a></li><li id="ue593f08a" data-lake-index-type="0"><span class="ne-text">打开</span><a href="https://www.bing.com/" data-href="https://www.bing.com/" target="_blank" class="ne-link"><span class="ne-text">https://www.bing.com/</span></a></li><li id="u30361e52" data-lake-index-type="0"><span class="ne-text">点击左下角CK </span></li><li id="ua759153f" data-lake-index-type="0"><span class="ne-text">Cookies到手</span></li></ul></details>
</div>
</body>
</html>

**Comment by twsec-pro** [comment]
这个有更详细的步骤吗？
![image](https://user-images.githubusercontent.com/58679784/233567365-3450b289-e38d-42f0-ae90-4eccff1f6bd8.png)

**Reply by araea** [reply]
直接用记事本打开，然后找到这两个词就行，后面有注释，按着改就行，也可以看一下第一个参考链接里的内容。
